0.10.4/alpine/Dockerfile

```diff
@@ -0,0 +1,77 @@
+FROM alpine:latest
+
+MAINTAINER João Fonseca <jfonseca@uphold.com>
+
+RUN adduser -S litecoin
+
+ENV GOSU_VERSION=1.7 \
+  GOSU_SHASUM="34049cfc713e8b74b90d6de49690fa601dc040021980812b2f1f691534be8a50  /usr/local/bin/gosu"
+
+RUN apk --no-cache add openssl \
+  && wget -O /usr/local/bin/gosu https://github.com/tianon/gosu/releases/download/${GOSU_VERSION}/gosu-amd64 \
+  && echo "${GOSU_SHASUM}" | sha256sum -c \
+  && chmod +x /usr/local/bin/gosu
+
+ENV BERKELEYDB_VERSION=db-4.8.30.NC
```

Comment:
Can we replace this with the [db-dev](https://pkgs.alpinelinux.org/package/v3.3/main/x86/db-dev) alpine package?